src/Stormpath/DataStore/DefaultDataStore.php

```diff
@@ -223,7 +223,7 @@ private function executeRequest($httpMethod, $href, $body = '', array $query = a
 
         $response = $this->requestExecutor->executeRequest($request);
 
-        $result = $response->getBody() ? json_decode($response->getBody()) : '';
+        $result = $response->getBody() ? json_decode($response->getBody()) : null;
```

Comment:
I made this change because the POST to /verificationEmails returns an empty body. It seems the resource factory is able to handle a `null` body, but not a body represented as an empty string `''`.